src/common/tvgMath.h

```diff
@@ -36,12 +36,12 @@
 #define mathMin(x, y) (((x) < (y)) ? (x) : (y))
 #define mathMax(x, y) (((x) > (y)) ? (x) : (y))
 
-
 bool mathInverse(const Matrix* m, Matrix* out);
-Matrix mathMultiply(const Matrix* lhs, const Matrix* rhs);
-void mathRotate(Matrix* m, float degree);
 bool mathIdentity(const Matrix* m);
-void mathMultiply(Point* pt, const Matrix* transform);
+void operator*=(Point& pt, const Matrix& m);
+Matrix operator*(const Matrix& lhs, const Matrix& rhs);
+void operator*=(Matrix& m, float degree);
```

Comment:
maybe could we add some comment here? imo this is counterintuitive that matrix * float is a rotation...

Reply:
let's revert it.